publish-to-gts/publish-to-gts.qml

```diff
@@ -0,0 +1,380 @@
+import QtQml 2.0
+import QOwnNotesTypes 1.0
+
+Script {
+    property string serverInstance;
+    property string authCode;
+    property string visibility;
+    property bool local_only;
+    property bool sensitive;
+    property string spoiler_text;
+    property string language;
+    
+    property variant settingsVariables: [
+        {
+        "identifier": "serverInstance",
+        "name": "Server instance",
+        "description": "Server instance you want to connect to (i.e.: example.org - no spaces, no protocol, no slashes)",
+        "type": "string",
+        "default": "example.org",
+        },
+        {
+        "identifier": "authCode",
+        "name": "Authentication Code",
+        "description": "Code returned by GtS after performing a successful authentication, if you paste it here you won't need to authenticate again until expiry",
+        "type": "string",
```

Comment:
Do you want to consider `string-secret` from #3150?

Reply:
You need to update `minAppVersion` too if you use the `string-secret`.

Reply:
Yes, the next iteration will be with string -secret.